cmd/neofs-adm/internal/modules/morph/container.go

```diff
@@ -104,6 +104,72 @@ func dumpContainers(cmd *cobra.Command, _ []string) error {
 	return os.WriteFile(filename, out, 0o660)
 }
 
+func listContainers(cmd *cobra.Command, _ []string) error {
+	filename, err := cmd.Flags().GetString(containerDumpFlag)
+	if err != nil {
+		return fmt.Errorf("invalid flag %s: %w", containerDumpFlag, err)
+	}
+
+	c, err := getN3Client(viper.GetViper())
+	if err != nil {
+		return fmt.Errorf("can't create N3 client: %w", err)
+	}
+
+	inv := invoker.New(c, nil)
+
+	nnsCs, err := c.GetContractStateByID(1)
+	if err != nil {
+		return fmt.Errorf("can't get NNS contract state: %w", err)
+	}
+
+	var ch util.Uint160
+	s, err := cmd.Flags().GetString(containerContractFlag)
```

Comment:
Missing flag returns default value, `err` here is for developer errors, or am I mistaken?